src/commcare_cloud/ansible/roles/devops_scripts/templates/db-tools.py.j2

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+import psycopg2
+import getopt
+import sys
+
+
+
+# Parse commandline options
+def usage():
+    """
+    Print the usage message for th script.
+    """
+    print """Usage:
+    list-all: To list all databases
+
+    """
+
+
+def list_all_database():
+    """
+    Prints the list of all database in PostgreSQL cluster.
+    """
+    try:
+        conn=psycopg2.connect("dbname='postgres'")
+    except:
+        print "Can not connect to database."
+    cur = conn.cursor()
+    try:
+        cur.execute("""SELECT datname FROM pg_database""")
+    except:
+        print "Can not select databases."
+
+    rows = cur.fetchall()
+    for row in rows:
+        db_name = row[0]
+        print(db_name)
+
+    conn.close()
+
+
+if __name__ == '__main__':
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], 'l', ['list-all'])
+    except getopt.GetoptError:
+        usage()
+        sys.exit(2)
+
+    if not opts:
+        usage()
+        sys.exit(2)
+
+# Parse GetOpts and execute the desired functionality
+    for opt, arg in opts:
+        if opt in ('-l', '--list-all'):
+            list_all_database()
+        else:
+            usage()
+            sys.exit(2)
```

Comment:
do we really need this script? Is there any reason we can't run the command directly instead of via a script? From the shell you can run: `psql -l -A -F, -t` which will output the list of databases in CSV format.

Reply:
Thanks for reviewing. I answered this in the first comment.